arts-university-bournemouth.csl

```diff
@@ -69,15 +70,14 @@
   </macro>
   <macro name="access">
     <group delimiter=" ">
-      <text term="online" prefix="[" suffix="]." text-case="capitalize-first"/>
+      <text term="online" prefix="[" suffix="]."/>
       <choose>
         <if variable="DOI">
           <text variable="DOI" prefix="https://doi.org/"/>
         </if>
         <else>
           <group delimiter=" ">
-            <text term="available at" text-case="capitalize-first"/>
-            <text variable="URL"/>
+            <text variable="URL" prefix="Available from: "/>
```

Comment:
@Dbelk, we can show you how, but it's better to change the English translation of the "available at" term than putting the text in a prefix.